server/router_utils.go

```diff
@@ -129,3 +130,60 @@ func readOrganizationID(writer http.ResponseWriter, request *http.Request) (type
 
 	return types.OrgID(organizationID), nil
 }
+
+// readClusterNames does the same as `readClusterName`, except for multiple clusters.
+func readClusterNames(writer http.ResponseWriter, request *http.Request) ([]types.ClusterName, error) {
+	clusterNamesParam, err := getRouterParam(request, "clusters")
+	if err != nil {
+		message := fmt.Sprintf("Cluster names are not provided %v", err.Error())
+		log.Println(message)
```

Comment:
I'm not sure that logs are needed here, or could you, please, provide more specific information in it?

Reply:
Please look at the original functions that I used as references: readClusterName and readOrganizationID. They have the exact same behavior.